### ISSUE TITLE
!ERROR=252  (java.lang.IllegalStateException: Date [1979-08-03] could not be parsed)

### ISSUE THREAD

**Issue by hyyan** [BasisHub/components#155]
In BBj 19.10 (nightly) the following snippet throws an Error where this used to work in previous BBj versions, what has been changed?

```java
use com.basiscomponents.db.DataRow
use java.sql.Types

dr! = new DataRow()
dr!.setFieldValue("DATE",java.sql.Types.DATE,"2013-02-01")
```

**Comment by DaAmore** [comment]
Hi @hyyan What Error is thrown?

**Comment by StephanWald** [comment]
!ERROR=252  (java.lang.IllegalStateException: Date [2013-02-01] could not be parsed)

**Comment by DaAmore** [comment]
Fixed in f79b5496341eaa01ba9cf8c86ba15d4194dc602f
